src/plugins/intel_gna/legacy/src/convert_function_to_cnn_network.cpp

```diff
@@ -1968,15 +1970,26 @@ CNNLayerCreator::CNNLayerCreator(const std::shared_ptr<::ngraph::Node>& node) :
                        });
 }
 
-CNNLayerPtr CNNLayerCreator::create() {
-    LayerParams attrs = {node->get_friendly_name(),
-                         node->description(),
-                         details::convertPrecision(node->get_output_element_type(0))};
-    if (creators.find(node->description()) != creators.end())
-        return creators[node->description()](node, params);
+CNNLayerPtr CNNLayerCreator::create(const std::shared_ptr<::ngraph::Node>& node_) {
+    node = node_;  // node used by node->visit_attributes(..) > AttributeVisitor::on_attribute(..)
```

Comment:
if you pass and use node_ everywhere field `node` of creator seems to be redundant and could be removed.
Thanks that you could make probably make `create` const due the fact no field would be modified.

Reply:
>if you pass and use node_ everywhere field node of creator seems to be redundant and could be removed.

it is needed by callbacks. that's why comment added to the code